hyperchain-docker-compose.yml

```diff
@@ -52,6 +53,17 @@ services:
       # We bind only to 127.0.0.1, so setting insecure password is acceptable here
       - POSTGRES_PASSWORD=notsecurepassword
   
+  pgadmin4:
+    image: dcagatay/pwless-pgadmin4:latest
+    ports:
+      - 8432:80
```

Comment:
love it - let's put it on the 15430 port (close to 15432 where postgres is).
and restrict to 127.0.0.1